content/news/release-20171014.md

```diff
@@ -0,0 +1,44 @@
+After three months of public playtesting we are pleased to finally declare "Mission Accomplished" on the new OpenRA release!  Release 20171014 makes some fundamental changes to the way that the OpenRA game engine works, plus a few similarly important gameplay changes to the default mods.
+
+The first big change will be visible as soon as you go to launch your favourite mod.  The in game mod chooser has been removed, and mods are now launched directly from your operating system.  This is part of a much larger set of changes to the game engine to support multiple OpenRA installations and the new [Mod SDK](https://github.com/OpenRA/OpenRAModSDK).
+
+<div style="text-align:center" markdown="1">
+<img src="/images/news/20171014-dmg.png" width="600" alt="New launchers">
+
+The mod chooser screen has been replaced by separate launchers for each mod.
+</div>
+
+The next big change is the new command bar at the bottom of the screen.  This new UI improves discoverability for several advanced unit commands and served as a focus for several gameplay changes related to unit stances and the Attack Move command – read through the in game button tooltips to find out more!
+
+<div style="text-align:center" markdown="1">
+<img src="/images/news/20171014-commandbar.png" width="600" alt="Unit command bar">
+
+A new bottom command bar is now available for issuing common orders and changing unit stances.
+</div>
+
+The final group of big changes is less obvious, but just as important: the way that buildings are targeted and take damage has been reworked (taking inspiration from the original Dune 2000) to account for the size of the building footprint.  This has a range of balance implications, for example Tanya and Commandos no longer need to run to the center of buildings to demolish them.  Explosion effects also look a lot better!
```

Comment:
Just wondered if we could emphasize more that explosions of warhead impacts look better (at first I thought this is about building explosions, but those didn't change?).

Reply:
I find that last sentence a little misleading, too, since we didn't change anything about the explosion graphics, only where they 'land'/trigger.

Reply:
The idea was that having them not all impact at the center of the actor looks a lot better, but I can see the confusion.  I may just delete this sentence.